app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.java

```diff
@@ -136,20 +137,42 @@ public void bindViewHolder(FlexibleAdapter<IFlexible> adapter, ConversationItemV
                 holder.dialogUnreadBubble.setText(R.string.tooManyUnreadMessages);
             }
 
-            if (conversation.isUnreadMention() || conversation.type == Conversation.ConversationType.ROOM_TYPE_ONE_TO_ONE_CALL) {
+            ColorStateList lightBubbleFillColor = ColorStateList.valueOf(
+                ContextCompat.getColor(context,
+                R.color.conversation_unread_bubble));
+            int lightBubbleTextColor = ContextCompat.getColor(
+                context,
+                R.color.conversation_unread_bubble_text);
+            ColorStateList lightBubbleStrokeColor = ColorStateList.valueOf(
+                ContextCompat.getColor(context,
+                R.color.colorPrimary));
+
+            if (conversation.type == Conversation.ConversationType.ROOM_TYPE_ONE_TO_ONE_CALL) {
                 holder.dialogUnreadBubble.setChipBackgroundColorResource(R.color.colorPrimary);
                 holder.dialogUnreadBubble.setTextColor(Color.WHITE);
+            } else if (conversation.isUnreadMention()) {
+                if (CapabilitiesUtil.hasSpreedFeatureCapability(userEntity, "direct-mention-flag")){
+                    if (conversation.getUnreadMentionDirect()) {
+                        holder.dialogUnreadBubble.setChipBackgroundColorResource(R.color.colorPrimary);
+                        holder.dialogUnreadBubble.setTextColor(Color.WHITE);
+                    } else {
+                        holder.dialogUnreadBubble.setChipBackgroundColor(lightBubbleFillColor);
+                        holder.dialogUnreadBubble.setTextColor(lightBubbleTextColor);
+                        holder.dialogUnreadBubble.setChipStrokeWidth(4.0f);
```

Comment:
From the screenshots I think the border could be thicker.

Reply:
@marcoambrosini can you have a look at the screenshots and what border size is the web UI going to use?

Reply:
stroke width 4.0
![grafik](https://user-images.githubusercontent.com/2932790/134319657-a623eee8-4f38-4013-af87-d5ee56d35773.png)

stroke width 5.0
![grafik](https://user-images.githubusercontent.com/2932790/134320206-07a1658d-9892-469e-967c-d7319d7225da.png)

stroke width 6.0
![grafik](https://user-images.githubusercontent.com/2932790/134319951-09d944ce-32e1-485f-979f-132bf31dd275.png)

Reply:
updated screenshots. now with stroke width 6.0